config/environments/development.rb

```diff
@@ -100,14 +100,14 @@
   # If you are using HTTP's basic authentication, you probably want to use this
   # option.
 
-  LOGOUT_REDIRECT = 'DEFAULT'
+  LOGOUT_REDIRECT = "DEFAULT"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.